Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -637,3 +561,17 @@ void EndJumpCheckDelay()
 	}
 	#endregion
 }
+
+public abstract class PhysicsState
+{
+	public abstract void Enter();
+	public abstract void Update();
+	public abstract void Exit();
+}
+
+public class DefaultState : PhysicsState
```

Comment:
We should rename this to DefaultPhysicsState. Since it inherits from Physics state it's not like we're going to reuse it for anything else.